x-pack/plugins/canvas/.storybook/webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = (_baseConfig, _env, config) => {
       include: /\.storybook/,
       loaders: 'babel-loader',
       options: {
-        presets: [require.resolve('@babel/preset-react')],
+        presets: [require.resolve('@kbn/babel-preset/webpack_preset')],
```

Comment:
Maybe we just want to extend the regex in this loader to support .ts files?

Reply:
@spalger I thought about that but it looks like that for `ts` there is also an additional loader `require.resolve('react-docgen-typescript-loader'),`